CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 Locks library change log
 ========================
 
+UNRELEASED
+==========
+
+  * ADDED: XCommon CMake support in examples
```

Comment:
Is this a change? xcommon has been removed.

Reply:
Good point; will fix.